packages/ui-transfer-components/lib/demo/demo-loading.component.ts

```diff
@@ -0,0 +1,34 @@
+import { css, html, unsafeCSS } from 'lit-element';
+import { RxLitElement } from 'rx-lit';
+import { Theme } from '@digita-ai/ui-transfer-theme';
+
+export class DemoLoadingComponent extends RxLitElement {
+
+  public message = 'Checking permissions...';
+
+  /**
+   * Renders the component as HTML.
+   *
+   * @returns The rendered HTML of the component.
+   */
+  render() {
+
+    return html`
+    <loading-component .message="${this.message}"></loading-component>
```

Comment:
Since the message is a string, it is cleaner to set it as an attribute (which automatically updates the corresponding property) than as the property itself.
 ```suggestion
    <loading-component message="${this.message}"></loading-component>
```

Reply:
Ah, thanks for the comment. That actually fixed the problem I commented above. Can now just add a normal loading component to the index.html without the demo component. Will remove the demo component.